### PR TITLE
use classNameBinding to repalce classNames in Ember.view

### PR DESCRIPTION
**Pull request by Cuiyansong**
Not modify ember-table-header-cell style and overwrite it on demo-app.

### CODE REVIEW

**Comment by jefflage** [comment]
flex doesn't work on all browsers.  but i think its ok in this case as someone can just override with their own style

File: addon/styles/addon.less [hunk 1]
```diff
@@ -461,8 +450,30 @@
   }
 
   &.sorted {
-    .ember-table-content-container .ember-table-content {
-      padding-right: 25px;
+    padding: 0;
+    position: relative;
+    // Override the jQuery sortable
```
Comment (jefflage) [inline addon/styles/addon.less]: i find it hard to believe that jQuery is doing this.  is maybe jqueryui or some plugin?

Comment (cyjia) [inline addon/styles/addon.less]: Yes, jqueryui.sortable.

Comment (jefflage) [inline addon/styles/addon.less]: did we add jqueryui or was it already there with base ember-table?

Comment (cyjia) [inline addon/styles/addon.less]: introduced by base ember-table.

**Pull request merged**